### PR TITLE
feat: Add skills page with searchable descriptions

### PR DESCRIPTION
**Pull request by brioja**
https://github.com/RedWrath5/wikigrisser-next/assets/34580058/a73f4f6c-26a6-494a-8ae2-3107f27abae6

Sort of a first pass. 3c skills are not included, it seems.

### CODE REVIEW

File: util/loaders/SkillsLoader.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ export class SkillsLoader extends Loader<SkillsMap> {
     let notDone = true;
 
     while (notDone) {
-      const skill: Skill = {
-        name: skillsSheat["A" + rowCounter]?.v || null,
+        const skill: Skill = {
+            name: typeof skillsSheat["A" + rowCounter]?.v === 'number' ? skillsSheat["A" + rowCounter]?.v.toString() : skillsSheat["A" + rowCounter]?.v || null,
```
Comment (brioja) [inline util/loaders/SkillsLoader.ts:22]: the skill 120% is being read in as a number

File: components/skill/SkillPage.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                setSearchText(event.target.value);
+              }}
+            />
+          </FormControl>
+        </div>
+      </div>
+
+      {/* Skills list */}
+      <BoundedColumn>
```
Comment (RedWrath5) [inline components/skill/SkillPage.tsx:41]: to make this center properly it needs a parent that is w-100

image

**Comment by RedWrath5** [comment]
image

this weird physical damage item is probably a data loading artifact. Also the mobile layout could be improved with centering as well as fitting more of the data on one line rather than seperate

**Comment by brioja** [comment]
Yeah, it's a loading artifact from the spreadsheet. I can fix it in another pull request.

File: components/skill/SkillPage.tsx (hunk 2 above)
Comment (brioja) [reply components/skill/SkillPage.tsx:42]: Fixed
image

**Comment by brioja** [comment]
Can I make mobile look prettier in another pull request? I'm not sure how to do it at the moment.

**Comment by RedWrath5** [comment]
That fine if you remove the link to the page from the navbar

**Comment by brioja** [comment]
> That fine if you remove the link to the page from the navbar

Done

**Comment by brioja** [comment]
Issues created for tracking unresolved tasks

https://github.com/RedWrath5/wikigrisser-next/issues/55
https://github.com/RedWrath5/wikigrisser-next/issues/54

**Pull request merged**